server.js

```diff
@@ -3,7 +3,7 @@ const express = require('express');
 const app = express();
 
 app.get('/ping', (req, res) => {
-    res.send('pong')
+    res.json({'message': 'pong'})
```

Comment:
Consider using a more descriptive message in the JSON response. This could make it easier for clients to understand the response. [medium]